helm/recommendations/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "recommendations.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "recommendations.name" . }}
+    helm.sh/chart: {{ include "recommendations.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: app
+    app.kubernetes.io/part-of: {{ include "recommendations.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: NodePort
```

Comment:
want to expose this to `api-gateway` instances, hence it would have to be a different type. Probably in conjunction with an `Ingress`.